unlock-app/src/components/interface/Account.tsx

```diff
@@ -62,7 +64,10 @@ export function Account() {
                 <button
                   className="px-2 py-1 text-gray-900 bg-gray-200 rounded"
                   type="button"
-                  onClick={deAuthenticate}
+                  onClick={() => {
+                    deAuthenticate()
+                    storageService.signOut()
```

Comment:
can we move`signOut` inside `deAuthneticate`?

Reply:
We can but I didn't want to do it in this PR be because might change the behavior in multiple places instead being isolated.